rclcpp_examples/test/test_executables_example.py.in

```diff
@@ -30,7 +31,20 @@ def test_executable():
         else:
             exit_handler = ignore_exit_handler
             exit_on_match = False
-        handler = create_handler(name, launch_descriptor, output_file, exit_on_match=exit_on_match)
+
+        filtered_prefixes = [
+            b'pid', b'rc',
```

Comment:
I think those lines are specific to launch itself? So tests should not need to know about them.